docs/python/editing.md

```diff
@@ -80,7 +80,7 @@ If autocomplete and IntelliSense aren't working for a custom module, check the f
 
 | Cause | Solution |
 | --- | --- |
-| The path to the python interpreter is incorrect | Check the `pythonPath` setting. Restart VS Code if you make a correction. |
+| The path to the python interpreter is incorrect | Make sure you selected a valid interpreter path by running the `Python: Select Interpreter` command (see [Environments](/docs/python/environments.md)). |
```

Comment:
For commands that are displayed in the VS Code UI like the Command Palette, use bold to format the text.
"by running the **Python: Select Interpreter** commmand"

Reply:
Thank you! Fixed it 😊